articles/api/authentication/_multifactor-authentication.md

```diff
@@ -0,0 +1,644 @@
+# Multifactor Authentication
+
+The Multifactor Authentication (MFA) API endpoints allow you to enforce MFA when users interact with [the Token endpoints](/api/authentication#get-token), as well programmatically enroll and manage user authenticators.
```

Comment:
- "programmatically" is redundant, you are already in the API docs so it is done programatically by definition
- `/api/authentication#get-token` should be `#get-token`